application-xpoll-test/application-xpoll-test-tests/src/test/it/org/xwiki/xpoll/test/ui/XPollTest.java

```diff
@@ -136,7 +136,7 @@ private CreatePagePage createPage(XPollHomePage xpollHomePage)
     {
         CreatePagePage createPage = xpollHomePage.createPage();
         createPage.getDocumentPicker().setTitle(pollName);
-        createPage.setTemplate("XPollCode.XPollTemplateProvider");
+        createPage.setTemplate("XPoll.XPollCode.XPollTemplateProvider");
```

Comment:
The [best practice](https://dev.xwiki.org/xwiki/bin/view/Community/ApplicationDevelopmentBestPractices#HPages) is to use the ``Code`` space. So it should be:

    XPoll.Code.XPollTemplateProvider